### ISSUE TITLE
[Bug Report] unsupported tunnel type while using shadowsocks with `simple-obfs`

### ISSUE THREAD

**Issue by oluceps** [daeuniverse/dae#369]
### Checks

- [X] I have searched the existing issues
- [X] I have read the documentation
- [X] Is it your first time sumbitting an issue

### Current Behavior

Using `shadowsocks` node with `simple-obfs` plugin reported error and nodes not appear in node list.
Besides this all nodes works fine.

### Expected Behavior

Works.

### Steps to Reproduce

1. Adding a node, e.g. 
```
ss://TEST@test.test.xyz:1023/?plugin=simple-obfs%3Bobfs%3Dhttp%3Bobfs-host%3D53a8633.bing.com
```

2. Start `dae`, following logs appeared.
```
Dec 19 19:02:42 kaambl dae[1439537]: level=warning msg="handlePkt: failed to GetOrCreate: unknown network unsupported tunnel type: simpleobfs+udp"

...

Dec 17 17:19:33 kaambl dae[779372]: level=debug msg="Connectivity Check Failed" err="unknown network unsupported tunnel type: simpleobfs+udp" network="udp6(DNS)" node=ss
```

And cannot access to website.

### Environment

- **Dae version (use `dae --version`)**: https://github.com/daeuniverse/dae/commit/89628f1f28c6344c7f948fcf380b99c48dfc4479
- **OS (e.g `cat /etc/os-release`)**: NixOS 24.05.20231211.a9bf124
- **Kernel (e.g. `uname -a`)**: 6.5.13
- **Others**:

### Anything else?

FWIW, `SIP002` URL scheme hadn't define the mode field and `SIP003` plugin seems doesn't support UDP transport, the error info may indicated this.

**Comment by oluceps** [comment]
Cannot be reproduced for the time being. Closed.

**Comment by fengguowudi** [comment]
time="Jun 02 02:59:33" level=warning msg="handlePkt: dial udp: lookup pop.air.3278.0tk8a3a1q4t94dler.com: i/o timeout"
time="Jun 02 02:59:33" level=warning msg="handlePkt: dial udp: lookup pop.air.3278.0tk8a3a1q4t94dler.com: i/o timeout"
time="Jun 02 02:59:33" level=warning msg="handleConn: failed to dial 142.250.69.164:443: [simpleobfs]: dial to pop.air.3278.0tk8a3a1q4t94dler.com:1052: dial tcp: lookup pop.air.3278.0tk8a3a1q4t94dler.com: i/o timeout"
time="Jun 02 02:59:33" level=warning msg="handleConn: failed to dial 142.250.69.164:443: failed to select dialer from group proxy (tcp6): no alive dialer"

**Comment by olicesx** [comment]
Closing as outdated: reported against a pre-v2.0 build/environment that has since been superseded by the v2.0.0 / #1099 datapath and control-plane rework, and no longer reproducible against current main. If this still reproduces on the latest release, please open a new issue with current-version logs. Housekeeping after #1099.

**Comment by olicesx** [comment]
Closing as outdated: reported against a pre-v2.0 build/environment that has since been superseded by the v2.0.0 / #1099 datapath and control-plane rework, and no longer reproducible against current main. If this still reproduces on the latest release, please open a new issue with current-version logs. Housekeeping after #1099.
